aikau/src/main/resources/alfresco/services/UploadService.js

```diff
@@ -388,6 +437,35 @@ define(["dojo/_base/declare",
       },
 
       /**
+       * This updates the [uploadHistory]{@link module:alfresco/services/UploadService#uploadHistory} with the
```

Comment:
Is this a valid link? uploadHistory is defined in functions & has no valid JSDoc as far as I can see.